GVFS/GVFS.FunctionalTests/Tests/GitCommands/AddStageTests.cs

```diff
@@ -32,12 +32,30 @@ public void StageBasicTest()
         }
 
         [TestCase, Order(3)]
+        [Category(Categories.Mac.M2)]
+        public void AddAndStageHardLinksTest()
+        {
+            if (!this.FileSystem.SupportsHardlinkCreation)

```

Comment:
This is not adding any coverage is AddStageTests are only run with `SystemIORunner`